src/mesh_tools.rs

```diff
@@ -333,6 +334,158 @@ mod tests {
         Geometry::from_triangle_faces_with_vertices_and_normals(faces, vertices, vertex_normals)
     }
 
+    pub fn cube_sharp_same_len(position: [f32; 3], scale: f32) -> Geometry {
```

Comment:
This is misformatted. Either format properly manually, or don't use `#[rustfmt::skip]`...

Also... is this a 1:1 copy of this function in `geometry.rs`? If so, import it instead. If not, document the differences.

Reply:
The function from `geometry.rs` will be eventually removed as it's only good for testing. I think it shouldn't end up being compiled into the release, so I put a copy of it into the test scope here. Therefore the 1:1 copy.

Reply:
Okay, agreed!

Reply:
This is still misformatted though!

Reply:
Isn't this misformatted again but in a different way?

Reply:
i don't think so. `cargo fmt` formats it this way

Reply:
I just meant that the other function in `geometry.rs` uses `rustfmt::skip` and the numbers are aligned nicely into columns, but this is also fine if it's okay with both of you.